### DISCUSSION TITLE
'octoprint-cli is not a valid command'

### DISCUSSION THREAD

**Discussion by HippoProgrammer** [UserBlackBox/octoprint-cli#33] [General]
**Describe the bug**
After installation, the command is not recognised on Windows or Mac.

**To Reproduce**
1. Try `pip3 install octoprint-cli`.
2. Run `octoprint-cli`.
3. You get an error message that the command does not exist.

**Expected behavior**
I would be able to access the command.

**Screenshots**
<img width="1083" alt="Screenshot 2024-07-08 at 20 16 30" src="https://github.com/hexadecimalDinosaur/octoprint-cli/assets/172101796/5ec07aaf-7552-494e-aa1d-902708ebbb84">

**Environment Information:**
 - OS: Windows 11 / MacOS Sonoma 14.5
 - Version: 3.3.2 / 3.3.2
 - Python Version: 3.12.4 / 3.10.2
 - Dependency versions: up to date
 - OctoPrint version: latest

**Additional context**
I have tested this on a `git clone` version and the same problem arises.
Testing on a random GitHub Codespace I used a few weeks ago worked then. 
Yes, `config.ini` is present. 

Have been testing this with @DrygithTheGM as part of our project. On both our computers (see above) it does not work. Help would be greatly appreciated.

**Comment by hexadecimalDinosaur** [comment]
This issue does not relate to the project code and is an error in how your Python environment is set up. See the highlighted error for details
![image](https://github.com/hexadecimalDinosaur/octoprint-cli/assets/36890802/9fb6e37a-a7ac-4623-8e7e-af6332184555)
Add the `bin` folder that pip installs scripts to into your user or system PATH variable.

**Comment by HippoProgrammer** [comment]
Thanks very much for your help.
